lua/nvim_lsp/kotlin_language_server.lua

```diff
@@ -0,0 +1,58 @@
+--- default config for gradle-projects of the
+--- kotlin-language-server: https://github.com/fwcd/kotlin-language-server
+---
+--- This server requires vim to be aware of the kotlin-filetype.
+--- You could refer for this capability to:
+--- 	https://github.com/udalov/kotlin-vim (recommended)
+--- 	Note that there is no LICENSE specified yet.
+
+local vim = vim	-- from global table
+local util = require 'nvim_lsp/util'
+local configs = require 'nvim_lsp/configs'
+--- kotlin-language-server supports multiple types of projects.
+--- The default works with single- and multi-module gradle projects.
+local function get_project_root()
+  return util.root_pattern("settings.gradle")
+end
+
+configs.kotlin_language_server = {
+  default_config = util.utf8_config {
+	filetypes = { "kotlin" };
+	root_dir = get_project_root();
+	log_level = vim.lsp.protocol.MessageType.Warning;
+  };
```

Comment:
`log_level` is the same as the default. So you don't need to redefine.
```suggestion

configs.kotlin_language_server = {
  default_config = util.utf8_config {
    filetypes = { "kotlin" };
    root_dir = util.root_pattern("settings.gradle");
  };
```